Automattic/jetpack
# [Paid Newsletters] Add the "Lower transaction fees" Post Publish Panel

Fixes #31302

## Proposed changes:

Implement the design in https://github.com/Automattic/jetpack/issues/31302



### Other information:


https://github.com/Automattic/jetpack/assets/7076981/7839ecfc-ac89-40f0-9fb5-378f8cf0a6c0


## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






1. Publish a newsletter post on WPCOM
2. Notice the post-publish panel titled "Lower transaction fees" is displayed when the WPCOM's commission is more than `0`
3. Publish a newsletter post on a Jetpack site
4. Notice the post-publish panel titled "Lower transaction fees" is never displayed
